Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM adoptopenjdk/openjdk15 as builder
+FROM adoptopenjdk/openjdk16 as builder
```

Comment:
better use the openj9 jvm because of lower ram usage

Reply:
there is unfortunately no openj9 image for arm64.